app/helpers/law_category_button_helper.rb

```diff
@@ -0,0 +1,32 @@
+# 법안 카테고리 버튼을 생성하는 헬퍼 메서드
+
+module LawCategoryButtonHelper
+    def law_category_button(tab, options = {})
+        # 법안 카테고리 정보
+        categories = {
+            "government-administration" => "🏛 정부·행정과 공공제도",
+            "legislative-judicial" => "⚖ 입법·사법과 선거제도",
+            "publicsecurity-nationaldefense" => "🚔 치안·사법과 국방",
+            "economy-finance" => "💰 경제·재정과 공정거래",
+            "industry-sciencetechnology" => "🔬 산업·과학기술과 정보통신",
+            "land-urbandevelopment" => "🏗 국토·도시개발과 환경",
+            "health-welfare" => "🏥 보건·복지와 안전망",
+            "education-culture" => "🎓 교육·문화와 관광",
+            "labor-humanrights" => "👥 노동·인권과 성평등",
+            "diplomacy-unification" => "🌍 외교·통일과 국제협력",
+            "disaster-climate" => "🆘 재난·기후와 원자력 안전"
+        }
```

Comment:
```suggestion
module LawCategoryButtonHelper
  CATEGORIES = {
    "government-administration" => "🏛 정부·행정과 공공제도",
    "legislative-judicial" => "⚖ 입법·사법과 선거제도",
    "publicsecurity-nationaldefense" => "🚔 치안·사법과 국방",
    "economy-finance" => "💰 경제·재정과 공정거래",
    "industry-sciencetechnology" => "🔬 산업·과학기술과 정보통신",
    "land-urbandevelopment" => "🏗 국토·도시개발과 환경",
    "health-welfare" => "🏥 보건·복지와 안전망",
    "education-culture" => "🎓 교육·문화와 관광",
    "labor-humanrights" => "👥 노동·인권과 성평등",
    "diplomacy-unification" => "🌍 외교·통일과 국제협력",
    "disaster-climate" => "🆘 재난·기후와 원자력 안전"
  }.freeze
```

모듈레벨 상수로 정의하고 .freeze를 통해 업데이트를 방지하면 좋을 것 같습니다.